lumicks/pylake/adjustments.py

```diff
@@ -47,19 +52,20 @@ def __init__(self, minimum, maximum, mode="absolute"):
             absolute_adjustment = lk.ColorAdjustment([50, 30, 20], [1000, 195, 95])
             stack.plot_correlated(file.force1x, channel="rgb", adjustment=absolute_adjustment)
         """
-        minimum, maximum = np.atleast_1d(minimum), np.atleast_1d(maximum)
+        minimum, maximum, gamma = (np.atleast_1d(x) for x in (minimum, maximum, gamma))
         self.mode = None if mode == "nothing" else mode
         if not self.mode:
             return
 
-        for bound in (minimum, maximum):
+        for bound in (minimum, maximum, gamma):
             if len(bound) not in (1, 3):
-                raise ValueError("Color value bounds should be of length 1 or 3")
+                raise ValueError("Color value bounds and gamma should be of length 1 or 3")
```

Comment:
Not sure whether it is necessary to have a different gamma for each color channel. But no harm to keep that option.

Reply:
Code-wise it was actually simpler to offer this feature than not. I also didn't really see any harm in allowing it.